packages/perseus/src/widgets/interactive-graphs/graphs/polygon.tsx

```diff
@@ -51,9 +51,7 @@ export const PolygonGraph = (props: Props) => {
                 points={[...points]}
                 color="var(--movable-line-stroke-color)"
                 svgPolygonProps={{
-                    strokeWidth: active
-                        ? "var(--movable-line-stroke-weight-active)"
-                        : "var(--movable-line-stroke-weight)",
```

Comment:
After discussions with our designer, we realized that other similar graphs do not have their lines become weighted when active. The line related graphs (linear, linear system, etc) become weighted because you can move the line by dragging it. Polygon and other graphs are dragged by clicking anywhere in the area filled with blue. When focusing any graph, the lines do not become weighted, just outlined.